### ISSUE TITLE
[BUG]: Many controls lack screen reader labels

### ISSUE THREAD

**Issue by cartertemm** [Mintplex-Labs/anything-llm#1022]
### How are you running AnythingLLM?

Local development

### What happened?

Currently, when screen reading software like [NVDA (non-visual desktop access)](https://github.com/nvaccess/nvda) is used to navigate the UI, it is difficult to deduce what many of the buttons and links do.

This is because they are denoted by icons, not text. It would be extremely beneficial for blind users to supply labels for UI controls that can supplement the icons.
With ARIA, these modifications wouldn't need to alter or inhibit the current layout in any way. The simplest improvements that I envision would be:

* Label the back button on the onboarding screen
* Label the continue button on the onboarding screen
* Label the social links, which are currently only represented by images
* Label some of the navigation items (general appearance, settings button)
* Indicate the currently selected workspace to screen readers, and mark up the relationship as a list container for quick navigation
* Indicate the currently selected thread to screen readers, and mark up the relationship as a list container for quick navigation
* Label the button that shows thread options
* Label copy and thumb up/down buttons in a chat
* Label stop generating button

### Are there known steps to reproduce?

I can provide more info if requested.

**Comment by timothycarambat** [comment]
🙏 ! Thank you for raising this (and i see the PR now too). Will review tomorrow
